test/tests/neml_simple/le_stress_variableOverwrite.i

```diff
@@ -0,0 +1,286 @@
+#
```

Comment:
Get rid of extra comment lines. The indenting for these files is a little off too.  Use 2 spaces for each indent level. You can use the `hit` tool in MOOSE to auto-indent.

Reply:
I cleaned up all of the indentation in the first commit.